.github/workflows/ci.yml

```diff
@@ -95,30 +95,5 @@ jobs:
         env:
           # https://docs.cypress.io/guides/references/advanced-installation#Skipping-installation
           CYPRESS_INSTALL_BINARY: 0
-        # Need to do a bunch of work to generate the locale typings 🙃
-      - name: Prepare type declarations
-        run: |
-          corepack yarn run build:lib
-          corepack yarn run build:companion
-          corepack yarn run build:locale-pack
```

Comment:
`build:locale-pack` is called anywhere anymore, should we remove it @Murderlon?

Reply:
How is that possible? We still need to build the locale 🤔